packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -171,6 +172,12 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public fixedHeader?: string;
 
+	/**
+	 * Determine if the footer will be displayed or not
+	 */
+	@Input()
+	public footer?: boolean;
```

Comment:
I don't think this input is needed because it will only depend on the `footer` property of the `ColumnProperties` right?

Reply:
Actually, I use it here line 556
`		if (changes["footer"]) {
			this.isFooterEnabled = coerceBooleanProperty(this.footer);
		}`

and then I tested in the html of the table to display the line : 
`		<ng-container *ngIf="isFooterEnabled">
			<tr mat-footer-row *matFooterRowDef="displayedColumns"></tr>
		</ng-container>`

Maybe it's not the best way to do it ?

Reply:
But as I explained in my other remark, the footer should be shown based on what the developer passes in the `ColumnProperties`...